salt/modules/boto3_route53.py

```diff
@@ -746,13 +814,13 @@ def get_resource_records(HostedZoneId=None, Name=None, StartRecordName=None,
         args.update({'PrivateZone': PrivateZone}) if PrivateZone is not None else None
         zone = find_hosted_zone(**args)
         if not zone:
-            log.error("Couldn't resolve domain name %s to a hosted zone ID.", Name)
+            log.error("Couldn't resolve domain name {} to a hosted zone ID.".format(Name))
```

Comment:
Please revert this back to how it was before, we use % subsitution in logging for good reason.